python/hello-world/requirements.txt

```diff
@@ -0,0 +1,12 @@
+enum34==1.1.6
```

Comment:
I know it's common practice to do `pip freeze > requirements.txt`, but for the sake of simplicity we always put the minimum set here. I would recommend that you make this:

```
gcloud[grpc]==0.14.0
```